### ISSUE TITLE
Fix port socket

### ISSUE THREAD

**Issue by hongzhidao** [nginx/unit#204]
Hi.

```
diff -r fd323ad9e24f src/nxt_port_socket.c
--- a/src/nxt_port_socket.c	Wed Dec 12 19:57:15 2018 +0300
+++ b/src/nxt_port_socket.c	Sun Dec 09 05:37:57 2018 -0500
@@ -179,6 +179,7 @@
     msg->link.prev = NULL;

     msg->buf = m->buf;
+    msg->share = m->share;
     msg->fd = m->fd;
     msg->close_fd = m->close_fd;
     msg->port_msg = m->port_msg;
```

Thanks.

**Comment by VBart** [comment]
@hongzhidao Have you observed any bugs related to uninitialized `msg->share` or it's just because it feels right to initialize the field?

**Comment by hongzhidao** [comment]
@VBart 
The ticket is related to the other two tickets.
https://github.com/nginx/unit/issues/206
https://github.com/nginx/unit/issues/207
The key point is that the receiver may receive incomplete messages.
Before I think it's very important 'port socket' should be robust, so I added some hard-code for testing.
And I found the memory does increase while high concurrency by 'webbench' tool.
Maybe this situation of 'received incomplete messages' need to be considering.
